src/gui/tray/ActivityItem.qml

```diff
@@ -17,6 +17,10 @@ MouseArea {
     property bool isTalkReplyOptionVisible: model.messageSent !== ""
     readonly property bool isCallActivity: model.objectType === "call"
 
+    property color headerColor: UserModel.currentUser.headerColor
+    property color themedHeaderColor: Theme.darkMode ? Qt.lighter(headerColor, 2)
```

Comment:
It's not quite clear looking at the two names which one is themed and which one isn't (since in fact both are themed) -- maybe `themedHeaderColor` should be `adjustedHeaderColor`?